src/python/WMCore/MicroService/MSUnmerged/MSUnmergedRSE.py

```diff
@@ -74,13 +82,17 @@ def buildMongoProjection(self, fullRSEToDB=False):
             mongoProjection.update({"files": True})
         return mongoProjection
 
-    def readRSEFromMongoDB(self, collection):
+    def readRSEFromMongoDB(self, collection, useProjection=False):
```

Comment:
Don't forget updating the docstring please.

Reply:
done